CARDS-2708 / CARDS-2726 : Brand updates to email templates

sdumitriu opened the pull request.

sdumitriu committed:
CARDS-2708 / CARDS-2726 : Brand updates to email templates

marta- commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PM-OO-IP/mailTemplates/InitialInvitation/bodyTemplate.header.html:
```diff
@@ -0,0 +1,74 @@
+      hr {
+        margin: 2em 0;
+        opacity: .3;
+      }
```
Remove this and replace line 46 with ` opacity: .3;`?

marta- commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/UHN-IC-ED/mailTemplates/ReminderNotification/bodyTemplate.footer.html:
```diff
@@ -1,7 +1,6 @@
     <div class="footer">
-      <hr/>
       <a href="https://www.uhn.ca/Integrated-Care">
```
This link no longer belongs here around the generic image. Can the IC footer.html files be removed, and the link moved to the email body around "The Integrated Care Team" signature line?

marta- commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PM-OO/mailTemplates/InitialInvitation/bodyTemplate.header.html:
So many copies of the header because of the PM logo...

Could we refactor a bit to separate the document head from the displayed page header and avoid having to maintain so many identical copies of the styling?

marta- replied:
Per discussion, the refactoring will be done in a subsequent iteration. We will temporarily be maintaining the copies meanwhile.

marta- commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PM-OO-IP/mailTemplates/InitialInvitation/bodyTemplate.footer.html:
```diff
@@ -1,5 +1,5 @@
     <div class="footer">
-      <hr/>
+      <img alt="" src="cid:PM-Footer.png"/>
```
```suggestion
      <img alt="Princess Margaret Cancer Centre" src="cid:PM-Footer.png"/>
```

marta- commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PM-OO-IP/mailTemplates/InitialInvitation/bodyTemplate.html:
```diff
@@ -1,3 +1,6 @@
+    <div class="header">
+      <img alt="UHN" src="cid:PM-Header.png"/>
```
```suggestion
      <img alt="Princess Margaret Cancer Centre, UHN" src="cid:PM-Header.png"/>
```

marta- commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/mailTemplates/bodyTemplate.footer.html:
```diff
@@ -1,7 +1,6 @@
-      <hr/>
       <a href="https://www.uhn.ca/corporate/AboutUHN/Quality_Patient_Safety/Pages/patient_values.aspx">
-        <img alt="" src="cid:PE-Logo.png"/>
+        <img alt="" src="cid:Footer.png"/>
```
What looks better when images from this sender are not accepted by the email client: no `alt` or some alt like "A healthier world"?

sdumitriu replied:
With `alt=""`, it looks like a barely noticeable dot.

sdumitriu committed:
CARDS-2708 / CARDS-2726: Brand updates to email templates
Use bigger images to avoid a fuzzy look
Make the header 50% width and left aligned

sdumitriu committed:
CARDS-2708 / CARDS-2726: Brand updates to email templates
Add alt text to some images

marta- commented on prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/mailTemplates/bodyTemplate.header.html:
```diff
@@ -14,25 +14,25 @@
+      }
       .header img {
-        max-width: 45% !important;
+          width: 50%;
```
Are we sure we want the image to always take up half the width? not min-width?

sdumitriu replied:
The image is already much bigger than the fill width, currently 850px compared to the 600px of the email body.

sdumitriu committed:
CARDS-2708 / CARDS-2726: Brand updates to email templates
Add the standard UHN header to the IC emails as well
